server/games/game.py

```diff
@@ -908,7 +908,7 @@ def to_dict(self):
                     player.login for player in self.players
                     if self.get_player_option(player.id, "Team") == team
                 ]
-                for team in self.teams
+                for team in self.teams if team
```

Comment:
Maybe do `is None` since this will also ignore the team if it’s set to `0`

Reply:
Yeah I can do that but as far as I know 0 would be an invalid value as well. But still would be good.

Reply:
Yea it shouldn’t come up in SupCom, but still nice to be robust. Also there is TAF which forked from here and occasionally merges our new changes in, and in TA maybe `0` is valid idk.